docs/src/documentation/05-development/04-migration-guides/01-v20.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: v20
+description: How to migrate to Orbit 20.0.0
+redirect_from:
+  - /migration-guides/v20/
+---
+
+# Orbit Migration Guide v20
+
+This migration guide focuses on the process of migrating from Orbit v19 to v20.0, as some breaking changes were introduced.
+With this guide, we aim to walk through all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortlessly.
+
+## Breaking changes
+
+### Required props in HorizontalScroll if `arrows` is true
+
+The `HorizontalScroll` component now requires `arrowLeftAriaLabel` and `arrowRightAriaLabel` props if `arrows` is true.
+
+**Before:**
+
+```jsx
+import HorizontalScroll from "@kiwicom/orbit-components/lib/HorizontalScroll";
+
+<HorizontalScroll arrows />;
+```
+
+**Now:**
+
+```jsx
+import HorizontalScroll from "@kiwicom/orbit-components/lib/HorizontalScroll";
+
+<HorizontalScroll arrows arrowLeftAriaLabel="Scroll left" arrowRightAriaLabel="Scroll right" />;
+```
+
+Make sure to provide translated strings for the `arrowLeftAriaLabel` and `arrowRightAriaLabel` props.
+
+To ease the migration, we provide a codemod to help you out, with some default generic translations.
+The codemod will inject FormattedMessage components with the default translations, in this case:
+
+```jsx
+// left arrow
+<FormattedMessage id="common.screenreader.arrow.scroll_left" defaultMessage="Scroll left" />
+// right arrow
+<FormattedMessage id="common.screenreader.arrow.scroll_right" defaultMessage="Scroll right" />
+```
+
+To run the codemod, use the following command:
+
+```bash
+npx jscodeshift -t https://raw.githubusercontent.com/kiwicom/orbit/master/packages/orbit-components/transforms/horizontalscroll-aria-labels.js --parser=tsx --extensions=ts,tsx <pathToYourCode>
+```
+
+Make sure to run prettier after running the codemod, as it might introduce some formatting changes.
+
+If necessary, feel free to adjust the translations to your needs.
```

Comment:
Just thinking, should we advise here what is the recommended tkey structure?

Reply:
Good point, will add.